gap/digraph.gi

```diff
@@ -1104,3 +1104,36 @@ InstallMethod(RandomTournament, "for a func and an integer",
 function(func, n)
   return RandomTournamentCons(func, n);
 end);
+
+InstallMethod(RandomLatticeCons, "for IsMutableDigraph and a pos int",
+[IsMutableDigraph, IsPosInt],
+function(filt, n)
+  local fam, rand_blist;
+
+  fam := [BlistList([1 .. n], [])];
+
+  while Length(fam) < n do
+    rand_blist := List([1 .. n], x -> Random([true, false]));
+    UniteSet(fam, List(fam, x -> UnionBlist(x, rand_blist)));
```

Comment:
I have no idea if this would make any difference but you could use `Set` instead of `List` here.

Reply:
`UniteSet(set, list)` is defined for a set and a list, so I do not think that having a `Set` instead of a `List` as the second argument changes much. In practice is a slightly slower with `Set` (presumably since it needs to reorder the list and then `UniteSet` does not speed up).

Reply:
Ok, that all makes sense.